commit cf40e2489b6d3b0c6297e4ea36c963007eac1748
Author: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
Date:   Sun Oct 19 07:28:15 2014 -0400

    add lock-obj header for or1k-unknown-linux-gnu
    
    * src/syscfg/lock-obj-pub.or1k-unknown-linux-gnu.h: new
    
    This architecture-specific header information was sourced from
    http://wannabuild.cmd.nu/fetch.php?pkg=libgpg-error&arch=or1k&ver=1.13-2&stamp=1407320768

diff --git a/src/syscfg/lock-obj-pub.or1k-unknown-linux-gnu.h b/src/syscfg/lock-obj-pub.or1k-unknown-linux-gnu.h
new file mode 100644
index 0000000..60eadab
--- /dev/null
+++ b/src/syscfg/lock-obj-pub.or1k-unknown-linux-gnu.h
@@ -0,0 +1,24 @@
+## lock-obj-pub.or1k-unknown-linux-gnu.h
+## File created by gen-posix-lock-obj - DO NOT EDIT
+## To be included by mkheader into gpg-error.h
+
+typedef struct
+{
+  long _vers;
+  union {
+    volatile char _priv[32];
+    long _x_align;
+    long *_xp_align;
+  } u;
+} gpgrt_lock_t;
+
+#define GPGRT_LOCK_INITIALIZER {1,{{0,0,0,0,0,0,0,0, \
+                                    0,0,0,0,0,0,0,0, \
+                                    0,0,0,0,0,0,0,0, \
+                                    0,0,0,0,0,0,0,0}}}
+##
+## Local Variables:
+## mode: c
+## buffer-read-only: t
+## End:
+##
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -56,6 +56,7 @@
         syscfg/lock-obj-pub.m68k-unknown-linux-gnu.h        \
         syscfg/lock-obj-pub.mips-unknown-linux-gnu.h        \
         syscfg/lock-obj-pub.mipsel-unknown-linux-gnu.h      \
+        syscfg/lock-obj-pub.or1k-unknown-linux-gnu.h        \
         syscfg/lock-obj-pub.powerpc-unknown-linux-gnu.h     \
         syscfg/lock-obj-pub.powerpc64-unknown-linux-gnu.h   \
 	syscfg/lock-obj-pub.powerpc64le-unknown-linux-gnu.h \
